src/app/plugins/checkclasses.plugin.ts

```diff
@@ -0,0 +1,74 @@
+import { Guild, GuildChannel } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+
+export class CheckClassesPlugin extends Plugin {
+  public name: string = 'Check Class';
+  public description: string = 'lists the classes someone is in';
+  public usage: string = 'checkclass <user>';
+  public pluginAlias = [];
+  public permission: ChannelType = ChannelType.Staff;
+
+  private _MAX_CHAR_LIMIT: number = 2000;
+  private _ALLOW_NUM: number = 3072; //ID for allowed permission overwrite
```

Comment:
is there a programmatic way to get this number from discord.js so it isnt so magical?

Reply:
The Deny ID is just 0, should I change it to deny ID and check the opposite?

Reply:
I am not sure that would work because:

a class channel is by default `DENY @. everyone` so there is no specific permission override to DENY that user (unless they *unregistered* but that's not important).

it's fine the way it is.. just a nitpick really :)